see/image_providers/os_glance.py

```diff
@@ -126,14 +125,24 @@ def _retrieve_metadata(self):
             raise FileNotFoundError(self.name)
 
     def _download_image(self, img_metadata, target):
+        partfile = '{}.part'.format(target)
+        if os.path.exists(partfile):
+            for image in sorted([img for img in self.glance_client.images.list()
+                                 if (img.id == self.name or
+                                     img.name == self.name)
+                                 and img.status == 'active'],
+                                key=lambda x: x.updated_at, reverse=True):
+                newtarget = os.path.join(os.path.dirname(target), image.id)
+                if os.path.exists(newtarget):
+                    return newtarget
+            raise FileNotFoundError()
```

Comment:
Is `FileNotFoundError` defined in SEE? Otherwise, it won't work on Python 2 as it was introduced in Python 3.

Reply:
It is redefined on a try/import statement https://github.com/F-Secure/see/blob/3d9c3a09d0511b8dd30f457545b9ddd46d4e21e0/see/image_providers/os_glance.py#L39